db/schema.rb

```diff
@@ -32,6 +32,15 @@
     t.index %w[submission_id], name: 'index_assets_on_submission_id'
   end
 
+  create_table 'notes', force: :cascade do |t|
+    t.string 'gravity_user_id'
```

Comment:
Looks like it's possible that you didn't re-dump the schema after editing the migration above potentially? Considering https://github.com/artsy/convection/pull/704/files#diff-f9057bc00c8c8baad7414f865649cac2R6

I would expect this line to read:

```ruby
t.string 'gravity_user_id, null: false
```

Reply:
Hmm, not sure how this happened but i did a `rake db:migrate:redo` and that fixed it. it is now in #707 and --i assume-- will fix itself there without the need for any other rollbacks or migrations.